books/free-programming-books-langs.md

```diff
@@ -516,6 +516,7 @@ Books on general-purpose programming that don't focus on a specific language are
 * [C++ Core Guidelines](https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md) - `edt.:` Bjarne Stroustrup, Herb Sutter
 * [C++ For Programmers](https://tfetimes.com/wp-content/uploads/2015/04/c-for-c-programmers.pdf) - JT Kalnay (PDF)
 * [C++ GUI Programming With Qt 3](https://ptgmedia.pearsoncmg.com/images/0131240722/downloads/blanchette_book.pdf) - Jasmin Blanchette, Mark Summerfield (PDF)
+* [C++ In Action: Industrial Strength Programming Techniques](http://www.worldcolleges.info/sites/default/files/C%2B%2B_In_Action.pdf) - Bartosz Milewski (PDF)
```

Comment:
It's published by Pearson, they don't even let the author distribute it. Also it's >25 years old. We won't bother with it..